managed/services/agents/jobs.go

```diff
@@ -33,16 +32,8 @@ import (
 	"github.com/percona/pmm/managed/models"
 )
 
-var (
-	// ErrRetriesExhausted is returned when remaining retries are 0.
-	ErrRetriesExhausted = errors.New("retries exhausted")
-
-	pmmAgentMinVersionForMongoLogicalBackupAndRestore  = version.Must(version.NewVersion("2.19"))
-	pmmAgentMinVersionForMySQLBackupAndRestore         = version.Must(version.NewVersion("2.23"))
-	pmmAgentMinVersionForMongoPhysicalBackupAndRestore = version.Must(version.NewVersion("2.31.0-0"))
-	pmmAgentMinVersionForMongoDBUseFilesystemStorage   = version.Must(version.NewVersion("2.32.0-0"))
-	pmmAgentMinVersionForMongoPITRRestore              = version.Must(version.NewVersion("2.32.0-0"))
```

Comment:
I can't find where all these version were moved?

Reply:
Version "constants" where moved on a higher level cuz we don't need them in jobs package anymore.

https://github.com/percona/pmm/blob/3e61da1ae0708bd6c8c8a5063cd18e99148edd27/managed/services/backup/compatibility_service.go#L31

Reply:
And for Mongo it was replaced with a single version cuz from this pr we cannot support older agents for Mongo backups.

Reply:
> And for Mongo it was replaced with a single version cuz from this pr we cannot support older agents for Mongo backups.

Can you remind me why?

Reply:
We're requesting for `mongod` and `pbm` binaries versions. Old agents know nothing about those. Actually the latest review requiest was about this behaviour. Is it ok if we just say that user needs upgrade agents to be able to make backups? Or we need some complex logic to support older agents?

Reply:
Usually we don't break compatibility that way. But backups in general not so stable feature yet.  Better ask @rnovikovP here.